### ISSUE TITLE
ARP storm

### ISSUE THREAD

**Issue by pekdz** [wildan2711/multipath#4]
I think this code doesn't install a group table into switch for ARP packet if there is a loop, which may cause severe ARP storm when the topology contains lots of loops(e.g., FAT Tree).

**Comment by wildan2711** [comment]
Yes, that may be true, I tested it on simple multipaths. Do you have an example case?

**Comment by pekdz** [comment]
> Yes, that may be true, I tested it on simple multipaths. Do you have an example case?

Yes, you can use this script to generate a fat tree topology. 
https://github.com/muzixing/fattree

I think simply install/update group table for ARP packet when each loop discovered/updated will solve this problem.
